ports/mimxrt/modmachine.c

```diff
@@ -48,6 +50,9 @@ STATIC const mp_rom_map_elem_t machine_module_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR_mem8),                MP_ROM_PTR(&machine_mem8_obj) },
     { MP_ROM_QSTR(MP_QSTR_mem16),               MP_ROM_PTR(&machine_mem16_obj) },
     { MP_ROM_QSTR(MP_QSTR_mem32),               MP_ROM_PTR(&machine_mem32_obj) },
+    { MP_ROM_QSTR(MP_QSTR_LED),                 MP_ROM_PTR(&machine_led_type) },
+    // class constants
+    { MP_ROM_QSTR(MP_QSTR_BOARD_LED_ID),        MP_ROM_INT(MACHINE_BOARD_LED) },
```

Comment:
I don't think this is needed.  Most peripherals (eg Pin, I2C, SPI, UART) require you to know what value to pass to the constructor to get the right peripheral.  Rather it should be part of the documentation for each board.